### PR TITLE
fix(CardStack): #37

### PR DESCRIPTION
**Pull request by DiMoNTD**

### CODE REVIEW

**Comment by DiMoNTD** [comment]
@LeoLeBras see the PR plz.
Don't know why build failed on Travis :(

**Comment by LeoLeBras** [comment]
Hey @DiMoNTD 🌈 ,

Thanks for this PR ! Travis fails due to this error :
capture d ecran 2017-10-11 a 11 58 37

You need to update this snapshot to pass `travis`.

File: modules/CardStack.js [hunk 1]
```diff
@@ -21,7 +21,12 @@ const buildNavigationState = (
   entries: Array<Location>,
   cards: Array<Card>,
 ): NavigationState<{}> => {
-  return entries.reduce(
+  // Find last route and reproduce actual route stack
+  // Fix > https://github.com/LeoLeBras/react-router-navigation/issues/37
+  const lastRouteIndex = entries.findIndex((entry) => entry.pathname === location.pathname)
+  const nextEntries = entries.slice(0, lastRouteIndex + 1)
```
Comment (LeoLeBras) [inline modules/CardStack.js]: It seems more appropriate to rename this var `initialEntries`

Comment (DiMoNTD) [reply modules/CardStack.js]: Thanks for review.
Agree. I Will fix it.

**Comment by DiMoNTD** [comment]
@LeoLeBras I have problems with run jest testing. I run it from Windows, and I have such Error:
```
FAIL  modules\__tests__\StackUtils-spec.js
  ● Console

    console.error node_modules\react-test-renderer\cjs\react-test-renderer.development.js:1973
      React caught an error thrown by Route. You should fix this error in your code. Consider adding an error boundary to your tree to customize error handling behavior.
      
      Error: Cannot find module 'Text' from 'react-native-implementation.js'
      
      The error is located at: 
          in Route
          in Router
....
```

And such Error everywhere that have react-native Components in test.

So I can't update snapshots.

**Comment by DiMoNTD** [comment]
@LeoLeBras I fixed problems with jest.. just ran it from macOS 🌈 😆 
So PR now ready for merge!

**Comment by LeoLeBras** [comment]
Awesome ! ❤️

**Pull request merged**